src/V3Randomize.cpp

```diff
@@ -367,6 +370,32 @@ class RandomizeMarkVisitor final : public VNVisitor {
             if (!classp->user1()) classp->user1(IS_RANDOMIZED);
             markMembers(classp);
         }
+        if (nodep->classOrPackagep()->name() == "std") {
+            for (AstNode* pinp = nodep->pinsp(); pinp; pinp = pinp->nextp()) {
+                AstArg* const argp = VN_CAST(pinp, Arg);
+                if (!argp) continue;
+                AstNodeExpr* exprp = argp->exprp();
+                while (exprp) {
+                    AstVar* randVarp = nullptr;
+                    if (AstMemberSel* const memberSelp = VN_CAST(exprp, MemberSel)) {
+                        randVarp = memberSelp->varp();
+                        exprp = memberSelp->fromp();
+                    } else {
+                        AstVarRef* const varrefp = VN_AS(exprp, VarRef);
+                        randVarp = varrefp->varp();
+                        exprp = nullptr;
+                    }
+                    UASSERT_OBJ(randVarp, nodep, "No rand variable found");
+                    AstNode* backp = randVarp;
+                    while (backp && (!VN_IS(backp, Class) && !VN_IS(backp, NodeModule))) {
+                        backp = backp->backp();
+                    }
+                    VN_IS(backp, Class) ? VN_AS(backp, Class)->user1(IS_STD_RANDOMIZED)
+                                        : VN_AS(backp, NodeModule)->user1(IS_STD_RANDOMIZED);
```

Comment:
This is doing same thing on both.  Also note Classes are NodeModules.
```suggestion
                    backp->user1(IS_STD_RANDOMIZED);
```